modules/core/test/webgl/classes/texture.spec.js

```diff
@@ -123,12 +123,18 @@ function testFormatCreation(t, glContext, withData = false) {
     for (let type of formatInfo.types) {
       format = Number(format);
       type = Number(type);
+      if (
+        formatInfo.dataFormat === GL.DEPTH_COMPONENT ||
+        formatInfo.dataFormat === GL.DEPTH_STENCIL
+      ) {
+        withData = false;
+      }
       const data = withData ? TEXTURE_DATA[type] || DEFAULT_TEXTURE_DATA : null;
       const options = Object.assign({}, formatInfo, {
         data,
         format,
         type,
-        mipmaps: format !== GL.RGB32F, // TODO: for some reason mipmap generation failing for RGB32F format
+        mipmaps: formatInfo.dataFormat === GL.RGBA, // Avoid mipmapping compressed formats
```

Comment:
Is `GL.RGBA` true for `GL.R` etc? Maybe a bit more comment here?

Reply:
Ah, this is left over from when the TEXTURE_FORMATS table was expanded. I'll revert.